### ISSUE TITLE
Accessibility: drop-down menus missing placeholder text

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-reporting#682]
#### Describe the task

Some drop-down menus are missing placeholder text, while others are populated, creating an inconsistent user experience. 

Add placeholder text to:

Register an operation: Operation info (page 1)
- [ ] Select operation to register for

Submit a report: General stationary combustion excluding line tracing:
- [ ] GSC unit type
- [ ] Fuel name
- [ ] Gas type
- [ ] Methodology

Submit a report: Fuel combustion by mobile equipment
- [ ] Fuel name
- [ ] Gas type
- [ ] Methodology

**Comment by patriciarussellCAS** [comment]
@NicoleGovvy does having placeholder text also mean that the user could type in the field and that would help search for the item on the list that they want? Or is that a totally separate thing? 
We've had a request for that option in dropdown fields, so folks can type what they want and the option pops up in the menu!

**Comment by NicoleGovvy** [comment]
@patriciarussellCAS the functionality looks like it already exists? I just tried typing in a couple drop-downs for register an operation and the menu options were displaying just what matched what I was typing. You could change the text to ‘Select or type [name of field].

**Comment by NicoleGovvy** [comment]
You could change the text to ‘Select or type [name of field].

From: "Patricia Russell (CAS)" ***@***.***>
Reply-To: bcgov/cas-reporting ***@***.***>
Date: Thursday, April 24, 2025 at 4:44 PM
To: bcgov/cas-reporting ***@***.***>
Cc: "Smith, Nicole C WLRS:EX" ***@***.***>, mention ***@***.***>
Subject: Re: [bcgov/cas-reporting] Accessibility: drop-down menus missing placeholder text (Issue #682)

[EXTERNAL] This email came from an external source. Only open attachments or links that you are expecting from a known sender.

[https://avatars.githubusercontent.com/u/94197323?s=20&v=4]patriciarussellCAS left a comment (bcgov/cas-reporting#682)<https://github.com/bcgov/cas-reporting/issues/682#issuecomment-2829079868>

@NicoleGovvy<https://github.com/NicoleGovvy> does having placeholder text also mean that the user could type in the field and that would help search for the item on the list that they want? Or is that a totally separate thing?
We've had a request for that option in dropdown fields, so folks can type what they want and the option pops up in the menu!

**Comment by patriciarussellCAS** [comment]
AH, ok. I don't think this was implemented in Reporting, or at least not consistently. It would be good to identify where they can do this and where they can't so we can make it the same across the app. Good to know we already have the functionality in Reg!

**Comment by NicoleGovvy** [comment]
@patriciarussellCAS I can note that as I'm working through the reporting screens for design discrepancies.
